canvas_modules/common-canvas/src/common-properties/properties-main/properties-main.jsx

```diff
@@ -87,6 +87,12 @@ class PropertiesMain extends React.Component {
 			showPropertiesButtons: true,
 			editorSize: editorSize
 		};
+		this.flyoutWidth = {
```

Comment:
I'm not seeing this being used.  Am I missing something?

Reply:
This is being used in updateRightFlyoutWidth to enforce width.